docs/DeveloperGuide.md

```diff
@@ -832,8 +834,36 @@ exploratory* testing.
     1. Resize the window to an optimum size. Move the window to a different location. Close the window.
 
     2. Re-launch the app by double-clicking the jar file.<br>
-       Expected: The most recent window size and location is retained.
-
+        Expected: The most recent window size and location is retained.
+
+
+## Student class tests
+
+### Adding a student
+1. Adding a student while all students are being shown
+
+   1. Prerequisites: List all students using the `liststu` command. Multiple students in the
+      list.
+   2. Test case: `addstu n/John Lim Jun Jie h/@johnlimjj e/johnlim@example.com` <br>
+      Expected: Student with the relevant details added into the list. Details of the added student shown in the status
+      message.
+   3. Test case: `addstu` <br>
+      Expected: No student added. Error details shown in the status message. Status bar remains the same.
+   4. Other incorrect addstu commands to try: `addstu n/John Lim Jun Jie`, 
+      `addstu n/John Lim Jun Jie h/@johnlimjj e/johnlimexample.com`
+
+### Editing a student
+1. Editing a student while all students are being shown
+
+   1. Prerequisites: List all students using the `liststu` command. Multiple students in the
+      list.
+   2. Test case: `editstu 1 n/Mary Doe` <br>
+      Expected: First student's name is edited to Mary Doe. Details of edited student shown in the status message.
```

Comment:
Would be be good to include "The rest of the details remain unedited." ?

Reply:
Added